modules/android/vsandroid_vcxproj.lua

```diff
@@ -211,19 +236,38 @@
 			elements = table.join(elements, {
 				android.debugInformation,
 				android.strictAliasing,
-				android.thumbMode,
 				android.fpu,
 				android.pic,
---				android.ShortEnums,
+				android.shortEnums,
+				android.cStandard,
+				android.cppStandard,
 			})
 			if _ACTION >= "vs2015" then
 				table.remove(elements, table.indexof(elements, vc2010.debugInformationFormat))
-				table.remove(elements, table.indexof(elements, android.thumbMode))
+
+				elements = table.difference(elements, {
```

Comment:
Why not just use `table.remove` as above?

Reply:
[Fixed](https://github.com/ifarbod/premake-core/commit/8c7ccc99d4f2d913f1bbd4e30e6063e6a7f4bf87).